pkg/apis/policy/v1beta1/zz_generated.deepcopy.go

```diff
@@ -108,6 +109,11 @@ func DeepCopy_v1beta1_PodDisruptionBudgetSpec(in interface{}, out interface{}, c
 		in := in.(*PodDisruptionBudgetSpec)
 		out := out.(*PodDisruptionBudgetSpec)
 		*out = *in
+		if in.MinAvailable != nil {
```

Comment:
It was not a pointer field before; we're changing the API (in the first commit) to allow specifying of one of "maxUnavailable" and "minAvailable"